tests/cases/webroot_test.py

```diff
@@ -42,6 +42,9 @@ def testAccessWebRoot(self):
         self.assertTrue('girder_app.min.js' in body)
         self.assertTrue('girder_lib.min.js' in body)
 
+        emailAddress = self.model('setting').get(SettingKey.CONTACT_EMAIL_ADDRESS)
```

Comment:
This is just going to get the default.

To ensure that changes to this setting are immediately respected, we should next set this setting to something totally different, then make another request to get the webroot, and ensure that the value is updated.